src/nupic/bindings/experimental.i

```diff
@@ -173,129 +186,152 @@ using namespace nupic;
 
     def activateCells(self,
                       activeColumns,
-                      prevActiveExternalCellsBasal,
-                      prevActiveExternalCellsApical,
+                      reinforceCandidatesExternalBasal=(),
```

Comment:
Ah, thanks. I felt dirty using "list". I didn't realize the word "sequence" excluded iterators -- it only includes types that aren't exhausted the first time you iterate over them. I'll definitely change it to "sequence".